ui/tests/e2e/test.js

```diff
@@ -39,6 +39,17 @@ describe("End-to-end", () => {
     expect(grayDiv).toBeTruthy();
   });
 
+  test("Export to Saturn With Facet", async () => {
```

Comment:
Let's name the 2 tests:

- Export to Saturn - no selected cohort
- Export to Saturn - selected cohort